packages/webcert/src/components/header/WebcertHeaderUser.tsx

```diff
@@ -24,34 +29,42 @@ const StyledSpan = styled.span`
     font-style: italic;
   }
 `
+interface Props {
+  changeLinkPointer?: boolean
+}
 
-const WebcertHeaderUser: React.FC = () => {
+const WebcertHeaderUser: React.FC<Props> = () => {
   const user = useSelector(getUser, shallowEqual)
   const userLinks = useSelector(getUserResourceLinks)
   const { ppHost } = useSelector(getConfig)
   const protectedUserApprovalKey = 'wc.vardperson.sekretess.approved'
   const showProtectedUserApprovalModal = user?.preferences?.[protectedUserApprovalKey] !== 'true' && user?.protectedPerson
   const privatePractitionerPortal = userLinks?.find((link) => link.type === ResourceLinkType.PRIVATE_PRACTITIONER_PORTAL)
+  const [isExpanded, setIsExpanded] = useState(false)
 
+  const handleClick = () => {
+    setIsExpanded(!isExpanded)
+  }
   const goToPrivatePractitionerPortal = () => {
     window.open(`${ppHost}?from=${window.location.href}`, '_blank')
   }
 
   const toString = (user: User): React.ReactNode => {
     return (
       <Wrapper>
-        <UserWrapper>
-          <span>{`${user.name} - ${user.role}`}</span>
-          {user.protectedPerson && (
-            <StyledSpan>
-              <ProtectedPersonDoctorModal />
-            </StyledSpan>
+        <ExpandableBoxWrapper onClick={handleClick} changeLinkPointer={!!privatePractitionerPortal}>
```

Comment:
Maybe a test can be added here to verify that it expands/closes when clicking on it